privcount/counter.py

```diff
@@ -136,6 +140,43 @@ def check_noise_weight_sum(noise_weight_sum, description="sum"):
         return False
     return True
 
+def get_noise_weight_default(noise_weight_config):
+    '''
+    Returns the default noise weight, if present in noise_weight_config.
+    Otherwise, returns None.
+    '''
+    return noise_weight_config.get(DEFAULT_NOISE_WEIGHT_NAME, None)
+
+def has_noise_weight_default(noise_weight_config):
+    '''
+    Returns True if noise_weight_config has a default noise weight.
+    Otherwise, returns False.
+    '''
+    return get_noise_weight_default(noise_weight_config) is not None
+
+def get_noise_weight(noise_weight_config, fingerprint):
+    '''
+    Return sthe noise weight for fingerprint, which can be None.
```

Comment:
Fixed in https://github.com/teor2345/privcount/tree/test-tor-source-v2

The chutney target uses TestingTorNetwork.
(I wanted to test against a real tor relay, not one with many modified settings.)
But if you want to use TestingTorNetwork, you can add `TestingTorNetwork 1` to the end of the run_test.sh command-line.